File: GCFinalProject/ClientApp/src/app/classes.ts
export class Rootobject {
    count: number;
    results: Result[];
    constructor(count: number, results: Result[]) {
    this.count = count;
    this.results = results;
    }
}

export class Result {
    index: string;
    name: string;
    url: string;
    constructor(index: string, name: string, url: string){
        this.index = index;
        this.name = name;
        this.url = url;
    }
}

File: GCFinalProject/Models/Classes.cs
namespace GCFinalProject.Models
{

    public class Rootobject
    {
        public int count { get; set; }
        public Result[] results { get; set; }
    }

    public class Result
    {
        public string index { get; set; }
        public string name { get; set; }
        public string url { get; set; }
    }

}
